suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -18,36 +18,43 @@ import {
 import { selectFiatCurrency } from '@suite-native/module-settings';
 import { useFormatters } from '@suite-common/formatters';
 
-type AccountTransactionListItemProps = {
+import { TransactionListItemIcon } from './TransactionListItemIcon';
+
+type TransactionListItemProps = {
     transaction: WalletAccountTransaction;
 };
 
-const transactionIconMap: Partial<Record<TransactionType, IconName>> = {
-    recv: 'receive',
-    sent: 'send',
+type TransactionTypeProperties = {
```

Comment:
You don't need type so detailed:
```
prefix: string,
sign: string,
signColor: Color,
```

We also have type `Color` which is exactly for usage like this